src/components/Auth/AuthDetail/index.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from './AuthDetail.module.scss';
+
+export default function AuthDetail({ name, first, second }:
+{ name :string, first:string, second:string }) {
```

Comment:
따로 규칙을 정해둔 건 아니지만, 가독성 측면에서는 props가 3개 이상 늘어나면 타입을 따로 지정해주는게 더 깔끔하게 보인다고 생각해요.
```suggestion
interface Props { 
  name :string;
  first:string;
  second:string;
}
export default function AuthDetail({ name, first, second }: Props) {
```

그리고 보니깐 헤더 문구를 지정하는 컴포넌트니, `phrase, heading`같은 표현으로 바꿔주는건 어떨까요?

Reply:
수정했습니다!